op3_ball_detector/src/ball_detector.cpp

```diff
@@ -69,6 +73,12 @@ BallDetector::BallDetector()
   nh_.param<int>("ellipse_size", detect_config.ellipse_size, params_config_.ellipse_size);
   nh_.param<bool>("filter_debug", detect_config.debug, params_config_.debug);
 
+  // nh_.param<std::string>("name", color_config.name, params_color_.name);
+  // nh_.param<int>("test_val", color_config.test_val, params_color_.test_val);
```

Comment:
What is the reason for commenting out these lines?  If we are submitting a pull request, we should be making sure to mark to take dead code out later if we want to merge it to master.